modules/auxiliary/scanner/ipmi/ipmi_dumphashes.rb

```diff
@@ -128,9 +128,12 @@ def run_host(ip)
         if sess.data.length < 8
           ipmi_status("Refused IPMI open session request")
           rakp = nil
-          break
+          sleep(5) if session_succeeded
```

Comment:
Can we add a new datastore option for this? Just incase folk want to wait less time. We can default this to 5 if need be, we'd just like to offer the configuration value. Potential naming convention: `SESSION_RETRY_DELAY`

Reply:
Added, plus I added also SESSION_MAX_ATTEMPTS, which's been internal constant as well.